lib/workers/bin/run_single_worker.rb

```diff
@@ -60,13 +56,13 @@
   create_options = {:pid => Process.pid}
   runner_options = {}
 
-  if ENV["QUEUE"]
-    create_options[:queue_name] = ENV["QUEUE"]
-    runner_options[:ems_id] = worker_class.ems_id_from_queue_name(ENV["QUEUE"]) if worker_class.respond_to?(:ems_id_from_queue_name)
+  if ENV["EMS_ID"].to_i > 0
+    create_options[:queue_name] = "ems_#{ENV['EMS_ID']}"
+    runner_options[:ems_id]     = ENV["EMS_ID"].to_i
   end
 
-  worker = if options[:guid]
-             worker_class.find_by!(:guid => options[:guid]).tap do |wrkr|
+  worker = if ENV["GUID"]
```

Comment:
@NickLaMuro Please review.  It seemed redundant to have a GUID env var and a `--guid` command line option to this script. I went with all ENV all the time.  Let me know if that removal is a problem.  I can convert and deprecate if needed.

Reply:
I am going to have to do some splelunking to figure out why I did that in the first place, but I think there was a reason.

Reply:
Just be sure to change --help to show what the expected ENV vars are